torch/jit/annotations.py

```diff
@@ -26,21 +31,23 @@ def __getattr__(self, name):
             raise RuntimeError("Module {} has no member called {}".format(self.name, name))
 
 
-class EvalEnv(object):
-    env = {
+class EvalEnv:
```

Comment:
nit: probably good to still have the python 3 style class definition

Reply:
You mean Python 2 style?

Reply:
I mean the new style class. nvm on this, python3 default on using new style class.